igvm_c/tests/igvm_test.c

```diff
@@ -0,0 +1,275 @@
+#include <CUnit/CUnit.h>
```

Comment:
Possible to wire this up in the github-actions yaml as well?

Reply:
I've added a new action that builds the C API and runs the unit tests, failing if any of the tests fail.